src/cpu.c

```diff
@@ -459,6 +463,24 @@ static void cpu_commit_one (int cpu_num, /* {{{ */
 	}
 } /* }}} void cpu_commit_one */
 
+static void cpu_commit_num_cpu (gauge_t num_cpu) /* {{{ */
+{
+//// static void submit_value (int cpu_num, int cpu_state, const char *type, value_t value)
```

Comment:
That's the comment I mentioned earlier. I think you mistakenly removed this comment instead:

```c
/* Commits the number of cores */
```